api/libretimeapi/views.py

```diff
@@ -141,9 +142,19 @@ class PreferenceViewSet(viewsets.ModelViewSet):
 class ScheduleViewSet(viewsets.ModelViewSet):
     queryset = Schedule.objects.all()
     serializer_class = ScheduleSerializer
-    filter_fields = ("starts", "ends", "playout_status", "broadcasted", "is_valid")
+    filter_fields = ("starts", "ends", "playout_status", "broadcasted")
     model_permission_name = "schedule"
 
+    def get_queryset(self):
```

Comment:
Now that the get_queryset is defined I though you had to remove the queryset field ? And also specify the base name in the urlpatterns list ? 

I am mainly asking for education.

Reply:
If you leave the queryset field, then you don't need to specify the base name. As I am extending the queryset instead of replacing it in get_queryset, I figured I should leave it and reference it here instead

Reply:
If you leave the queryset field, then you don't need to specify the base name. As I am extending the queryset instead of replacing it in get_queryset, I figured I should leave it and reference it here instead